llvm/test/Bitcode/compatibility.ll

```diff
@@ -2102,7 +2104,8 @@ define float @nofpclass_callsites(float %arg) {
 ; CHECK: attributes #48 = { allocsize(1,0) }
 ; CHECK: attributes #49 = { nosanitize_bounds }
 ; CHECK: attributes #50 = { allockind("alloc,uninitialized") }
-; CHECK: attributes #51 = { builtin }
+; CHECK: attributes #51 = { nosanitize_realtime }
+; CHECK: attributes #52 = { builtin }
```

Comment:
Why did I do 51 instead of 52? These orderings are very specific, it seems. I could not get them to play nicely if I appended this, so I put it at 51.

It seems like it compiles a bunch of little functions and checks the output is sane. All of the rest of them seem to be copy/paste but builtin is different somehow. This may be the reason it is last and should remain last